src/ILCompiler.ManagedAssembliesBuildTasks/src/ILCompiler.ManagedAssembliesBuildTask.csproj

```diff
@@ -0,0 +1,25 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" Condition="'$(IsProjectNLibrary)' != 'true'" />
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>ILCompiler</RootNamespace>
+    <AssemblyName>ILCompiler.ManagedAssembliesBuildTask</AssemblyName>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <OutputPath>$(BaseOutputPath)$(OSPlatformConfig)/tools</OutputPath>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="ComputeManagedAssemblies.cs" />
+  </ItemGroup>
+  <ItemGroup>
+  <PackageReference Include="Microsoft.Build.Framework">
+    <Version>15.3.0-preview-000388-01</Version>
```

Comment:
Can we depend on stable version of the packages?

https://www.nuget.org/packages/Microsoft.Build.Framework/